force-app/main/default/classes/Rollups.cls

```diff
@@ -0,0 +1,187 @@
+public with sharing class Rollups {
```

Comment:
Let's rename this to RollupService and have our Trigger call this service vs the ServiceDeliveryService and remove all knowledge of the RollupsService from the ServiceDeliveryService.